modules/wri_media/wri_media.install

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * @file
+ * Install and update hooks for wri_media.
+ */
+
+/**
+ * Use the oembed_lazyload display mode for videos.
+ */
+function wri_media_update_9301() {
+
+  // Add the new field configs.
```

Comment:
You need to enable the `oembed_lazyload` module before this will work. Let's add that to this update hook, and add it as a requirement in the wri_media.info.yml file.

Also, can you change this to 9303 with the assumption that https://github.com/wri/wri_sites/pull/68 will be merged first.